src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ILocalFunctionStatement.cs

```diff
@@ -0,0 +1,309 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
Done

---
In reply to: [121547310](https://github.com/dotnet/roslyn/pull/20177#discussion_r121547310) [](ancestors = 121547310)